src/schema/fair.ts

```diff
@@ -26,6 +28,19 @@ import ShowSort from "./sorts/show_sort"
 import { allViaLoader } from "lib/all"
 import { FairArtistSortsType } from "./sorts/fairArtistSorts"
 
+const FollowedContentType = () =>
```

Comment:
Curious, why the defer here? If the issue is referring to the `Artist` or `Partner` types and circular/diamond dependencies, you can do move that to just the `fields` part:  `fields: () => { ...`.

Reply:
Ah, that was exactly the issue, but your solution is cleaner, thanks! :)